lib/msal-common/src/error/ClientAuthError.ts

```diff
@@ -142,6 +142,10 @@ export const ClientAuthErrorMessage = {
     tokenRefreshRequired: {
         code: "token_refresh_required",
         desc: "Cannot return token from cache because it must be refreshed. This may be due to one of the following reasons: forceRefresh parameter is set to true, claims have been requested, there is no cached access token or it is expired."
+    },
+    userTimeoutReached: {
+        code: "user_timeout_reached",
```

Comment:
should we mention for device code polling to clarify use case?

Reply:
I agree that makes the error message more clear, I have added it @sameerag

Reply:
Great!